localnet/build_and_run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# Does a cargo build and then recreates the localnet using the new binaries.
+# Usage: ./localnet/build_and_run.sh
+
+cargo build && ./localnet/recreate.sh -d && cd bouncer && ./setup_for_test.sh
```

Comment:
So it doesn't actually change your directory
```suggestion
cargo build && ./localnet/recreate.sh -d && (cd bouncer && ./setup_for_test.sh)
```